keyboards/melgeek/z70ultra/config.h

```diff
@@ -18,12 +18,20 @@
 
 #include "config_common.h"
 
+
+
 /* COL2ROW, ROW2COL*/
 #define DIODE_DIRECTION COL2ROW
 
 /* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
 #define DEBOUNCE 3
 
+/* disable these deprecated features by default */
+#define _CHIBIOS_RT_CONF_VER_6_1_
+#define _CHIBIOS_HAL_CONF_VER_7_1_
+//#define NO_ACTION_MACRO
+//#define NO_ACTION_FUNCTION
+
```

Comment:
Please remove. None of these should go here, and the no action defines are deprecated.